### PR TITLE
feat: include smooth page to page transition

### PR DESCRIPTION
**Pull request by geekyahmed**
I included a new feature that provides a smooth page to page transition whenever the web app is loaded. I made use of [VuePageTransition](https://orlandster.github.io/vue-page-transition/ ) library to achieve this.

### CODE REVIEW

**Comment by ivelin** [comment]
Looks like a visual effect for smooth page transition. However it breaks the CI build. Can you try to fix that?

**Comment by geekyahmed** [comment]
> Looks like a visual effect for smooth page transition. However it breaks the CI build. Can you try to fix that?

Seen. I just fixed the codes and made another pull request. Hope you don't mind looking at the new pull request..

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "codecov": "^3.7.0",
     "compression-webpack-plugin": "^4.0.0",
+    "fs-vacuum": "^1.2.10",
```
Comment (ivelin) [inline package.json:20]: @geekyahmed Do we need this dependency on fs-vacuum? Where is it used within Ambianic UI?  

The [docs state](https://www.npmjs.com/package/fs-vacuum) its purpose is to:
"Remove the empty branches of a directory tree, optionally up to (but not including) a specified base directory. Optionally nukes the leaf directory."

**Comment by geekyahmed** [comment]
I found out that it was what cause the build to fail. So I had to install in so that the build will work properly.

**Comment by ivelin** [comment]
> I found out that it was what cause the build to fail. So I had to install in so that the build will work properly.

I see. Is this a dev dependency then or a production dependency?

**Comment by geekyahmed** [comment]
It is kind of a production dependency. Nothing major I think..

**Comment by ivelin** [comment]
Why do you think we need it in production if its only used during build?

> It is kind of a production dependency. Nothing major I think..

**Pull request merged**